pkg/operator/controller/dns_status.go

```diff
@@ -110,6 +123,11 @@ func computeDNSDegradedCondition(oldCondition *operatorv1.OperatorCondition, clu
 	}
 
 	if len(degradedReasons) != 0 {
+		// if the last status was set to false within the last statusTransitionAllowedDeltaTimeInSeconds, skip the new update
```

Comment:
`statusTransitionAllowedDeltaTimeInSeconds` should be `transitionUnchangedToleration`.